### PR TITLE
[FRONT-90] Add profile modal for logout and settings

### PR DESCRIPTION
**Pull request by WrappedSuperClass**
### Description of issue
Somewhere in the Sidebar, eg by clicking on the profile pic, you should get a dropdown with the options to Log Out or Settings, which will then later (not part of this task) lead to the Settings page.

### How I implemented
I used the Menu commponent and added a divider 
Fixed the path of the button of the sidebar

### Other

### CODE REVIEW

File: src/components/pure/Sidebar/index.tsx [hunk 1]
```diff
@@ -16,7 +23,39 @@ const Sidebar = ({ footer, ...props }: SidebarProps) => (
           <span className="mr-2 text-18 font-medium">{props.title}</span>
         </Link>
 
-        <div className="h-[30px] w-[30px] rounded-[5px] bg-gray-600" />
+        <Menu as="div" className="relative inline-block text-left">
+          <div>
+            <div className="" />
```
Comment (ArnoClaude) [inline src/components/pure/Sidebar/index.tsx:28]: What is this empty div for? Also, if you don't have any class names, you can just omit the className attribute :)

File: src/components/pure/Sidebar/index.tsx [hunk 2]
```diff
@@ -16,7 +23,39 @@ const Sidebar = ({ footer, ...props }: SidebarProps) => (
...
+
+            <Menu.Button className="flex h-[30px] w-[30px] flex-row items-center rounded-[5px] border border-gray-800 bg-gray-850 px-2 py-1.5 text-14 transition-colors duration-100 hover:bg-gray-700">
+              <AiFillSmile color="#858699" />
```
Comment (ArnoClaude) [inline src/components/pure/Sidebar/index.tsx:31]: Please use our brand colors from the `tailwind.config.js`

File: src/components/pure/Sidebar/index.tsx [hunk 3]
```diff
@@ -16,7 +23,39 @@ const Sidebar = ({ footer, ...props }: SidebarProps) => (
...
+            </Menu.Button>
+          </div>
+
+          <TransitionMenuItems>
+            <Menu.Items className="absolute right-0 z-10 mt-2 w-44 origin-top-right rounded-md bg-gray-700 shadow-lg ring-1 focus:outline-none">
```
Comment (ArnoClaude) [inline src/components/pure/Sidebar/index.tsx:36]: I would not make the width fixed. Just let the dropdown take up as much width as it needs :)

image

**Review by ArnoClaude — CHANGES_REQUESTED** [review]
Great job, Lukas! @WrappedSuperClass I left some minor improvement suggestions :)

File: src/components/pure/Sidebar/index.tsx (hunk 1 above)
Comment (WrappedSuperClass) [reply src/components/pure/Sidebar/index.tsx]: i don't know, tbh :)

I deleted it

File: src/components/pure/Sidebar/index.tsx (hunk 2 above)
Comment (WrappedSuperClass) [reply src/components/pure/Sidebar/index.tsx]: fixed!

File: src/components/pure/Sidebar/index.tsx (hunk 3 above)
Comment (WrappedSuperClass) [reply src/components/pure/Sidebar/index.tsx]: fixed!

**Review by kigadye — APPROVED** [review]

**Pull request merged**